content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+        and servers;
+
+    -   To maintain the official website, wics.uwaterloo.ca;
+
+    -   To be the authority on matters regarding usage of systems.
+
+7.  The duties of the Big CSters Director are:
+
+    -   To maintain the Big CSters community through useful and
+        inclusive programming, such as workshops and discussion circles;
+
+    -   To run the Big CSters mentoring program, holding an absolute
+        minimum of two events per term;
+
+    -   To administer the CSters mailing list.
+
+8.  The duties of the Diretor of Publicity are:
+
+    -   To publicize events and other endeavours of the committee
+        through physical and electronic means;
+
+    -   To design all media circulated by the committee including:
+        posters, social media profiles and the official website.
+
+# Procedures #
+
+Any changes to procedures must be proposed with at least one week's
+notice, and ratified at an official meeting of the undergraduate
+committee.
+
+## Policy ##
+
+1.  Code of Conduct\
+    The Code of Conduct outlines expected behaviour for all members of
+    the community at any event affiliated with the undergraduate
+    committee. Any member of the committee may prpose an amendment to
+    the CoC. The committee must ratify any proposed amendments to the
+    Code of Conduct.
+
+2.  Machine Usage Agreement\
+    The Machine Usage Agreement outlines expected conduct when using any
+    machines maintained by the undergraduate committee. The machine
+    usage agreement is maintained by the systems administrator. Any
+    proposed changes must be first ratified by wics-sys, followed by the
+    entire committee.
+
+3.  Office Poliy\
+    Office Policy outlines expected behaviour in the WiCS Undergraduate
+    Committee office, M3 1003. The office policy is maintained by the
+    Office Manager. Any proposed changes must be ratified by the
+    committee.
+
+## Succession Procedure ##
+
+1.  No later than the end of the first week of the fourth month of the
+    current term, a call for committee members should be announced on
+    the website and sent out to all affiliated mailing lists.
+
+2.  New volunteers caan apply to be on the committee for the following
+    term, and the current committee will select applicants for
+    interviews. A new volunteer shall be anyone who has never served on
+    the committee before, or has not served on the committee for 4 or
+    more terms. Interviewers should be chosen by the current committee.
+    If an interviewer has a conflict of interest with the interviewee,
+    an alternate interviewer will be chosen. In any case, if no
+    interviewer is explicitly chosen, the current Chair shall conduct
+    interviews. The interviews are intended to welcome the new member to
+    the committee, give them background on the committee's activities,
+    and help them find a role for the next term. The current committee
+    will ultimately select the comoposition of the committee for next
```

Comment:
comoposition -> composition

But they don't really select the composition, they select the members, so I think "members" should be used instead